security/pkg/pki/ca/controller/secret.go

```diff
@@ -113,7 +113,7 @@ func NewSecretController(ca ca.CertificateAuthority, core corev1.CoreV1Interface
 	return c
 }
 
-// Run starts the SecretController until stopCh is closed.
+// Run starts the SecretController until a value is sent to stopCh.
```

Comment:
What value does the stopCh expect? And how to send that value?

Reply:
An empty struct would suffice `struct{}{}`. Whoever is calling the Run can send the value to `stopCh` so that the underlying controllers are stopped. See original doc https://github.com/kubernetes/client-go/blob/master/tools/cache/controller.go#L97:8